assets/scss/local.scss

```diff
@@ -84,4 +84,11 @@ Used for table columns containing statuses which use the govuk-tag.
       white-space: nowrap;
     }
   }
+}
+
+/**
+Used to remove padding from GDS classes.
+*/
+.no-padding {
```

Comment:
I think you can use `govuk-!-padding-0` from GDS reading https://design-system.service.gov.uk/styles/spacing/#overriding-spacing